### PR TITLE
XS✔ ◾ Rules to better reporting services - Combine duplicate rules

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ A conversation with Adam - related email, subject "SSW Rules - Combine duplicate rules"

> 2. What was changed?

✏️ The duplicate rule was deleted

> 3. Did you do pair or mob programming (list names)?

✏️No

### CODE REVIEW

File: rules/when-to-use-reporting-services/rule.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-related:
-- customization-do-you-know-which-version-of-sql-reporting-services-and-visual-studio-you-are-using
-redirects: []
+created: 2013-07-02T08:25:41.0000000Z
+authors:
+- title: Adam Cogan
+  url: https://ssw.com.au/people/adam-cogan
+related: []
+redirects:
+- when-to-use-reporting-services
```
Comment (tiagov8) [inline rules/when-to-use-reporting-services/rule.md:13]: Hi @jeoffreyfischer 

This redirect seems to be the very same as the URI

- [ ] Please fix

**Comment by tiagov8** [comment]
Hey @jeoffreyfischer 

You also need to remove the duplicate from the category https://ssw.com.au/rules/rules-to-better-reporting-services/

- [ ] Please aciton

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/when-to-use-reporting-services/rule.md (hunk 1 above)
Comment (jeoffreyfischer) [reply rules/when-to-use-reporting-services/rule.md]: Done - the correct redirect has been used

**Comment by jeoffreyfischer** [comment]
@tiagov8 I have also updated the category file as you requested.

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**